frontend/server/src/main/java/org/pytorch/serve/openapi/OpenApiUtils.java

```diff
@@ -37,6 +37,7 @@ public static String listApis(ConnectorType type) {
     private static void listInferenceApis(OpenApi openApi) {
         openApi.addPath("/", getApiDescriptionPath("apiDescription", false));
         openApi.addPath("/ping", getPingPath());
+        openApi.addPath("/v1/models/{model_name}:predict", getPredictionsPath(false));
```

Comment:
@abishekchiffon Please add some comments on why the /v1/modelsxxx api is needed

Reply:
Added the comments